lasagne/init.py

```diff
@@ -8,14 +14,48 @@
 
 
 class Initializer(object):
+    """Initializer class
+
+    The :class:`Initializer` class represents a weight initializer used
+    to initialize weight parameters in a neural network layer. It should be
+    subclassed when implementing new types of weight initializers.
+
+    """
     def __call__(self, shape):
+        """
+           Subclasses of :class:`Layer` class use this method when they
+           initialize weight parameters.
+        """
         return self.sample(shape)
 
     def sample(self, shape):
+        """
+        Sample should return a theano.tensor of size shape and data type
+        theano.config.floatX.
+
+        Parameters
+        -----------
+            shape : tuple or int
+                Integer or tuple specifying the size of the returned.
+                matrix.
+            returns : theano.tensor
+                 Matrix of size shape and datatype theano.config.floatX.
```

Comment:
Indented by four spaces too much. `returned.` should be `returned` (without dot). `datatype` should probably be `dtype`.